doc/nestml_language/neurons_in_nestml.rst

```diff
@@ -70,6 +70,69 @@ It is equivalent if either both `inhibitory` and `excitatory` are given, or neit
    * - ``inhibitory``
      - ... should be negative. It is added to the buffer with non-negative magnitude :math:`-w`.
 
+The input ports can also be defined as a vector. For example,
+
+.. code-block:: nestml
+
+   parameters:
+    TWO integer = 2
+   end
+
+   input:
+     foo[TWO] pA <- spike
```

Comment:
Do we define what happens when the value of ``TWO`` is written to with a different number?

Reply:
Do you mean to say something like `TWO = 3`?

Reply:
Added a CoCo to check if the size of the vector is an integer.